From 72b250c04f543d4eeda06b32e699444b15cac5cc Mon Sep 17 00:00:00 2001
From: "ying.zhang" <ying.zhang22455@nxp.com>
Date: Sat, 17 Dec 2016 00:39:28 +0800
Subject: [PATCH 226/226] mtd:spi-nor:fsl-quadspi:Enable fast-read for
 LS1088ARDB

Add fast-read mode for LS1088ARDB board.

Signed-off-by: Yuan Yao <yao.yuan@nxp.com>
Integrated-by: Jiang Yutang <yutang.jiang@nxp.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1088a-rdb.dts |    2 ++
 arch/arm64/boot/dts/freescale/fsl-ls1088a.dtsi    |    1 +
 2 files changed, 3 insertions(+)

--- a/arch/arm64/boot/dts/freescale/fsl-ls1088a-rdb.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1088a-rdb.dts
@@ -91,6 +91,7 @@
 		compatible = "spansion,m25p80";
 		#address-cells = <1>;
 		#size-cells = <1>;
+		m25p,fast-read;
 		spi-max-frequency = <20000000>;
 		reg = <0>;
 	};
@@ -99,6 +100,7 @@
 		compatible = "spansion,m25p80";
 		#address-cells = <1>;
 		#size-cells = <1>;
+		m25p,fast-read;
 		spi-max-frequency = <20000000>;
 		reg = <1>;
 	};
--- a/arch/arm64/boot/dts/freescale/fsl-ls1088a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1088a.dtsi
@@ -294,6 +294,7 @@
 		interrupts = <0 25 0x4>; /* Level high type */
 		clocks = <&clockgen 4 3>, <&clockgen 4 3>;
 		clock-names = "qspi_en", "qspi";
+		fsl,qspi-has-second-chip;
 	};
 
 	pcie@3400000 {
